### DISCUSSION TITLE
GUI window show nothing(blank), is driver too old or is there other issue?

### DISCUSSION THREAD

**Discussion by skydig** [emilk/egui#1257] [Q&A]
as title,
display card info is as attached picture
![image](https://user-images.githubusercontent.com/27293515/154399863-8dd9ec7c-ea6a-421b-8dc3-c651f8d06498.png)

**Comment by emilk** [comment]
Have you tried the glow examples at https://github.com/grovesNL/glow ?

Did you try the glium backend?

**Reply by skydig** [reply]
I have not tried glow example, just run egui_lib_app by default under windows7.
how to compile and try glium backend?
thanks.
